talks/views.py

```diff
@@ -242,4 +249,9 @@ def deleteReply(request, reply_pk):
             replyObj.delete()
         
         return redirect(url)
-        
\ No newline at end of file
+
+def searchBarRedirect(request):
+    talk_name = request.GET.get('talk-name-input')
+    talkObj = get_object_or_404( Talk,name__icontains=talk_name)
```

Comment:
You should not be using `get_object_or_404` with a non `unique` query parameter like `name_icontains`. This can match multiple objects, and would throw an error, since `get` isn't meant for fetching multiple `objects`. The `talk` name is unique but this query won't return `unique` items. Switch this to use  a `filter` function